includes/Feed/FeedDataExporter.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+namespace SkyVerge\WooCommerce\Facebook\Feed;
+
+defined( 'ABSPATH' ) || exit;
+
+use SkyVerge\WooCommerce\Facebook\Feed\FeedProductFormatter;
+
+/**
+ * Class FeedDataExporter
+ *
+ * @since 2.6.0
+ */
+class FeedDataExporter {
+
+	/**
+	 * Various feed utilities.
+	 *
+	 * @var FeedProductFormatter Various feed utilities.
+	 */
+	protected $product_formatter;
+
+	/**
+	 * Cached attributes variants.
+	 *
+	 * @since 2.6.0
```

Comment:
> Every function, hook, class, and method should have a corresponding @since version associated with it (more on that below).

From https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#since-section-changelogs